pkg/common/cns-lib/volume/if.go

```diff
@@ -0,0 +1,21 @@
+package volume
+
+import (
+	"context"
+
+	"github.com/vmware/govmomi/vim25/types"
+
+	cnsvsphere "sigs.k8s.io/vsphere-csi-driver/v2/pkg/common/cns-lib/vsphere"
+)
+
+// TaskManagerIf provides methods to start and modify a listView object to monitor CNS tasks.
+// NewTaskViewManager satisfies TaskManagerIf and should be initialized to use the interface methods
+type TaskManagerIf interface {
+	AddTaskToListView(ctx context.Context, taskMoRef types.ManagedObjectReference, ch chan Result) error
+	RemoveTaskFromListView(ctx context.Context, taskMoRef types.ManagedObjectReference) error
+	ResetVirtualCenter(virtualCenter *cnsvsphere.VirtualCenter)
```

Comment:
This PR handles multi-vc FSS as well.
See this block of code, https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/csi/service/vanilla/controller.go#L216-L225. It will end up creating a separate listView instance for each VC.